src/pages/utilities/gestures.md

```diff
@@ -0,0 +1,208 @@
+---
+previousText: ''
+previousUrl: ''
+nextText: 'Platform'
+nextUrl: '/docs/utilities/platform'
+---
+
+# Gestures
+
+## Overview
+
+Ionic Gestures is a utility that allows developers to build custom gestures and interactions for their application in a platform agnostic manner. Developers do not need to be using a particular framework such as React or Angular, nor do they even need to be building an Ionic app! As long as developers have access to `@ionic/core`, they will have access to all of Ionic Gestures.
+
+Building complex gestures can be time consuming. Other libraries that provide custom gestures are often times too heavy handed and end up capturing mouse or touch events and not letting them propagate. This can result in other elements no longer being scrollable or clickable.
+
+## Basic Gestures
+
+### Usage
+
+```javascript
+let p = document.querySelector('p');
+const gesture = createGesture({
+  el: document.querySelector('.rectangle'),
+  onMove: (detail) => { onMove(detail); }
+})
+
+gesture.enable();
+
+const onMove = (detail) => {
+  const type = detail.type;
+  const currentX = detail.currentX;
+  const deltaX = detail.deltaX;
+  const velocityX = detail.velocityX;
+    
+  p.innerHTML = `
+    <div>Type: ${type}</div>
+    <div>Current X: ${currentX}</div>
+    <div>Delta X: ${deltaX}</div>
+    <div>Velocity X: ${velocityX}</div>
+  `
+}
+```
+
+In this example, our app listens for gestures on the `.rectangle` element. When a gesture movement is detected, the `onMove` function is called, and our app logs the current gesture information.
+
+<docs-codepen user="ionic" slug="xxKBYdL"></docs-codepen>
```

Comment:
I believe this Codepen has the wrong slug, it looks like an animation codepen

Reply:
My last reply could be due to the weirdness I'm seeing mentioned above

Reply:
Yeah I think you're right. The CodePen is a gesture example: https://codepen.io/ionic/pen/xxKBYdL